### PR TITLE
Coverity fixes

### PR DESCRIPTION
**Pull request by zrax**
Some low-hanging fruit Coverity fixes.  Some of these may impact behavior, and so should be tested.

### CODE REVIEW

**Comment by Hoikas** [comment]
How in the hell did this game ever work? This obviously looks amazing, but I'll do some testing just to make sure...

File: Sources/Plasma/PubUtilLib/plSDL/plStateVariable.cpp [hunk 1]
```diff
@@ -839,6 +839,7 @@ bool plSimpleStateVariable::IConvertFromShort(plVarDescriptor::Type newType)
             delete [] fS;
             fI = newI;
         }
+        break;
```
Comment (Hoikas) [inline Sources/Plasma/PubUtilLib/plSDL/plStateVariable.cpp]: :trollface:

**Comment by Hoikas** [comment]
:+1: -- book sharing still works

**Comment by Hoikas** [comment]
Interpreting radio silence to mean no one else has a problem.

**Comment by Deledrius** [comment]
:+1:

**Pull request merged**